e2e/web/fixture.ts

```diff
@@ -55,17 +56,23 @@
 // This new "test" can be used in multiple test files, and each of them will get the fixtures.
 export const test = newBase.extend<MyFixtures & MyOptions>({
   gameToResume: [ALMOST_FINISHED_GAME, { option: true }],
+  profileSetting: [{}, { option: true }],
 
   page: async ({ page }, use) => {
     await page.goto("");
     await use(page);
   },
   // Loads a game from local storage and navigates to resume the game.
-  resumeGame: async ({ page, gameToResume }, use) => {
+  resumeGame: async ({ page, gameToResume, profileSetting }, use) => {
+    await page.evaluate((gameToResume: JSON) => {
+      window.localStorage.setItem("active_game", JSON.stringify(gameToResume));
+    }, gameToResume as JSON);
+    if (profileSetting) {
+      await page.evaluate((profileSetting: JSON) => {
+        window.localStorage.setItem("profile", JSON.stringify(profileSetting));
+      }, profileSetting as JSON);
+    }
```

Comment:
We don't need to cleanup the test data as playwright does this for us. Also, since this is used for testing there isn't much value in encasing with a try / catch block.